src/includes/getting-started-install/dotnet.mdx

```diff
@@ -1,16 +1,19 @@
-Install the **NuGet** package:
+Install the **NuGet** package to add the Sentry dependency:
 
-```shell
-# Using Package Manager
-Install-Package Sentry
+```powershell {tabTitle:Package Manager}
+Install-Package Sentry -Version {{ packages.version('sentry.dotnet') }}
+```
 
-# Or using .NET Core CLI
-dotnet add package Sentry
+```shell {tabTitle:.NET Core CLI}
+dotnet add package Sentry -v {{ packages.version('sentry.dotnet') }}
+```
 
-# Or using Paket
-paket add Sentry
+```shell {tabTitle:Paket CLI}
+paket add Sentry --version {{ packages.version('sentry.dotnet') }}
 ```
 
-**Using .NET Framework prior to 4.6.1?**
+<Note>
+
+You can <PlatformLink to="/usage/#ignoring-errors">ignore exceptions</PlatformLink> by their type when initializing the SDK.
```

Comment:
Not sure it's worth adding this in the getting started. Makes sense to have it in the "Configuration" section.
Users won't need this information when they are first trying out the SDK.

Reply:
I will move it. Thanks.